diff --git a/plugins/python/uwsgiplugin.py b/plugins/python/uwsgiplugin.py
--- a/plugins/python/uwsgiplugin.py 2012-02-13 18:26:37.000000000 +0100
+++ b/plugins/python/uwsgiplugin.py 20122012-03-08 16:04:58.000000000 +0100
@@ -20,11 +20,8 @@
     else:
         try:
             LDFLAGS.append("-L%s" % sysconfig.get_config_var('LIBDIR'))
-            os.environ['LD_RUN_PATH'] = "%s" % (sysconfig.get_config_var('LIBDIR'))
         except:
             LDFLAGS.append("-L%s/lib" % sysconfig.PREFIX)
-            os.environ['LD_RUN_PATH'] = "%s/lib" % sysconfig.PREFIX
-
 
     version = sysconfig.get_config_var('VERSION')
     try:

